build-system/tasks/performance/measure-documents.js

```diff
@@ -30,6 +35,8 @@ function requirePuppeteer_() {
   puppeteer = require('puppeteer');
 }
 
+const TIMEOUT_MS = argv.set_timeout || 2000;
```

Comment:
How about moving the default timeout to config.json?

Reply:
Good call!